src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -822,10 +822,9 @@ internal void WriteInformation(InformationRecord record, bool overrideInquire =
                         }
                     }
                 }
-
-                if (!record.Tags.Contains("PSHOST") && (preference != ActionPreference.SilentlyContinue))
+                    
+                if (record.Tags.Contains("PSHOST") || (preference != ActionPreference.SilentlyContinue))
                 {
-                    // Only transcribe informational messages here. Transcription of PSHost-targeted messages is done in the InternalUI.Write* methods.
```

Comment:
Seems new comment about 'Inquire' and PSHost-targets will be useful.

Reply:
Most important change is that transcription of `Write-Host` and `Write-Information` is handled in the same place now.
The part about fixing `Inquire` has only historical value and does not introduce any new information about current behavior.
Anyway, take a look at my comment and let me know if you want it stated differently as English is not my native language.